tests/xharness/Microsoft.DotNet.XHarness.iOS.Shared/Utilities/PlistExtensions.cs

```diff
@@ -1,8 +1,17 @@
 ﻿using System.IO;
+using System.Runtime.CompilerServices;
```

Comment:
needed?

Reply:
Yes, that's where ConditionalWeakTable is.